### PR TITLE
Consolidate UG v1.3

### PR DESCRIPTION
**Pull request by Ferusel**
Does not close any specific issue.

This PR is meant to consolidate some changes after merging #258 and #261.

Currently blocked by:
- #227 (Unsure how to call stats command)
- #228 (Missing updated screenshots of FoodRem)

### CODE REVIEW

File: docs/_ug/About.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FoodRem is an Inventory Management System that empowers small food and beverage (F&B) restaurant managers to manage inventory and obtain insights from inventory data. As a restaurant manager, leverage upon FoodRem's Inventory Management System, where you can manage your inventory during your daily operations. Utilize FoodRem's flexible tagging system to help you organize your inventory according to your business needs. Finally, streamline your business decisions by deriving insights from your inventory usage through FoodRem's statistics, keeping track of vital data such as food wastage.
```
Comment (RichDom2185) [inline docs/_ug/About.md:1]: Missing lint ignore rule.

```suggestion
<!-- markdownlint-disable-file first-line-h1 -->
FoodRem is an Inventory Management System that empowers small food and beverage (F&B) restaurant managers to manage inventory and obtain insights from inventory data. As a restaurant manager, leverage upon FoodRem's Inventory Management System, where you can manage your inventory during your daily operations. Utilize FoodRem's flexible tagging system to help you organize your inventory according to your business needs. Finally, streamline your business decisions by deriving insights from your inventory usage through FoodRem's statistics, keeping track of vital data such as food wastage.
```

File: docs/_ug/HowToUseUserGuide.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Thank you for choosing FoodRem! We are delighted to have you as a user and aim to serve you well.
```
Comment (RichDom2185) [inline docs/_ug/HowToUseUserGuide.md:1]: Ditto on lint ignore.

```suggestion
<!-- markdownlint-disable-file first-line-h1 -->
Thank you for choosing FoodRem! We are delighted to have you as a user and aim to serve you well.
```

File: docs/_ug/Features.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+There are two core features that FoodRem provides: The Inventory Management System, and Inventory Analysis.
```
Comment (RichDom2185) [inline docs/_ug/Features.md:1]: Ditto on lint ignore.

```suggestion
<!-- markdownlint-disable-file first-line-h1 -->
There are two core features that FoodRem provides: The Inventory Management System, and Inventory Analysis.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -5,112 +5,27 @@ show-toc: true
...
-### Placeholders
-
-Placeholders are words in UPPER_CASE to show you what parameters you can supply to a command.
-
-{% include_relative _ug/Placeholders.md %}
-
-### Trying your First Command
 
-{% include_relative _ug/TryingFirstCommand.md %}
+{% include_relative _ug/UsingFoodRem.md %}
```
Comment (RichDom2185) [inline docs/UserGuide.md:28]: Why was this deleted?

**Review by RichDom2185 — CHANGES_REQUESTED** [review]
Overall LGTM with some changes requested, some of which I'll resolve and I'll await your reply for the remaining ones. Thanks for the refactoring!

File: docs/UserGuide.md (hunk 4 above)
Comment (bryanljx) [reply docs/UserGuide.md:28]: It was moved into the UsingFoodRem file

```
### Trying your First Command

{% include_relative _ug/TryingFirstCommand.md %}

```

Comment (RichDom2185) [reply docs/UserGuide.md:28]: Oops sorry missed that, thanks!

**Review by RichDom2185 — APPROVED** [review]
LGTM with a minor style nitpick, thanks for the refactoring!

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks for the refactor!
